### PR TITLE
Clarify & provide example of the validation return

### PR DESCRIPTION
**Pull request by cjvandyk**
In order to spare other developers from spending hours hunting this down, I've clarified that the validation token is sent in the querystring.  I've also provided an example of the return code for reference.

## Category

- [x] Content fix
- [ ] New article

## Related issues

- fixes #issuenumber
- partially #issuenumber
- mentioned in #issuenumber

## What's in this Pull Request?

The original article was not clear with regard to the validation token.  This change clarifies that the validation token is sent in the querystring.  It also provides an example of the return code for reference.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[d9ac3ed](https://github.com/cjvandyk/sp-dev-docs/commits/d9ac3ed60f62fca22fb7001f88098272f185fec6)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/apis/webhooks/lists/create-subscription.md](https://github.com/cjvandyk/sp-dev-docs/blob/patch-3/docs/apis/webhooks/lists/create-subscription.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/apis/webhooks/lists/create-subscription?branch=pr-en-us-8422) | [Details](#user-content-5e852a51e0dc87398f42aad8561cb63c-e6c7a511)

<a id="5e852a51e0dc87398f42aad8561cb63c-e6c7a511"></a>
### [docs/apis/webhooks/lists/create-subscription.md](https://github.com/cjvandyk/sp-dev-docs/blob/patch-3/docs/apis/webhooks/lists/create-subscription.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202209081251126714-8422/BuildReport?accessString=2d82178db925186feb52f024d05aa2bd2fa1a0701d74ec552ff2a3cb380afa7a).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by cjvandyk** [comment]
Hey Vesa,

I don't know how to fix that.  Its referring to line 2 which I did not touch.
Thoughts?

File: docs/apis/webhooks/lists/create-subscription.md [hunk 1]
```diff
@@ -83,9 +83,15 @@ Content-Type: application/json
 
 ## URL validation
 
-Before a new subscription is created, SharePoint sends a request with a validation token in the body of the request to the service URL provided. Your service must respond to this request by returning the validation token.
+**IMPORTANT - Before a new subscription is created, SharePoint sends a request with a validation token in the querystring of the request to the notification URL provided. Your service must respond to this request by returning the validation token. If your service fails to validate the request in this way, the subscription is not created.**
```
Comment (andrewconnell) [inline docs/apis/webhooks/lists/create-subscription.md:86]: Please change this to use the MSDocs format for an important callout. it shoudl be a block quote in the format of:

```markdown
> [!IMPORTANT]
> markdown text goes here
```

File: docs/apis/webhooks/lists/create-subscription.md [hunk 2]
```diff
@@ -83,9 +83,15 @@ Content-Type: application/json
 
 ## URL validation
 
-Before a new subscription is created, SharePoint sends a request with a validation token in the body of the request to the service URL provided. Your service must respond to this request by returning the validation token.
+**IMPORTANT - Before a new subscription is created, SharePoint sends a request with a validation token in the querystring of the request to the notification URL provided. Your service must respond to this request by returning the validation token. If your service fails to validate the request in this way, the subscription is not created.**
 
-If your service fails to validate the request in this way, the subscription is not created.
+### Example
+
+```
```
Comment (andrewconnell) [inline docs/apis/webhooks/lists/create-subscription.md:90]: Please include the language of this code in the snippet... like:

```javascript
// code goes here
```

**Review by andrewconnell — CHANGES_REQUESTED** [review]
Please address feedback in the review so we can get these changes merged in.

Also... 

When you submit a change to a doc, please update the `ms.date` front matter property to the format `MM/DD/YYYY` to reflect the date of the change.

**Review by andrewconnell — APPROVED** [review]

**Comment by VesaJuvonen** [comment]
Learn Build status updates of commit _[12127ed](https://github.com/cjvandyk/sp-dev-docs/commits/12127ed2ca01b7150b6710059d9ce00a4c6f3a2c)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-110b667c)

<a id="110b667c"></a>
###
 - **[Error: CannotMergeCommit]** `````Cannot merge commit 12127ed2ca01b7150b6710059d9ce00a4c6f3a2c in branch patch-3 of repository https://github.com/cjvandyk/sp-dev-docs into branch main (commit 09f6f74e1f3916f649eda745b6ef7b79fda66432). Please follow this documentation: https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/ to use git.exe to resolve you content conflicts locally and then push to remote.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202210201129501448-8422/BuildReport?accessString=9b2cb37dcdefed3ddc83171a3e0c6bc9882a2db29c3a7eb371baf199bab81dad).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Learn Build status updates of commit _[cd21f50](https://github.com/cjvandyk/sp-dev-docs/commits/cd21f50d87cb440d2909c77c738edcc98b06e5a9)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-b6d4673d)

<a id="b6d4673d"></a>
###
 - **[Error: GitBranchDeletedOrForcePushed]** 
`````
Cannot sync git repo to specified commit because ref +refs/pull/8422/merge has been deleted or has been force pushed.
 If this is pull request, please ensure it is mergeable (no merge conflict) and open (not closed).
`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202210201131297293-8422/BuildReport?accessString=dfdc8b5f6657f9929f2677f0a1b95bd8be670c8dcf3a11f411fc1d4737933aed).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Pull request merged**